utilities/cheyenne/launch-dask-scheduler.sh

```diff
@@ -18,5 +18,8 @@ source activate pangeo
 
 SCHEDULER=/glade/scratch/$USER/scheduler.json
 rm -f $SCHEDULER
-mpirun --np 6 dask-mpi --nthreads 6 --memory-limit 22e9 --interface ib0 \
-    --local-directory /glade/scratch/$USER --scheduler-file=$SCHEDULER
+mpirun --np 9 dask-mpi --nthreads 4 \
+    --memory-limit 12e9 \
```

Comment:
The 109GB limit is genuine?  Should we be more conservative here?  (I'm hoping the answer is "no")

Reply:
These nodes have 128GB total RAM and 109GB available. 9x12=108 so I think we'll be okay, provided that the nannies never let a task run over 12GB.

Reply:
See https://github.com/matthewsucar/userspace_oom for details on the nannies.
My understanding (not guaranteed to be correct) is that it cares only about total utilization.